### PR TITLE
Documentation for screenshot tests

### PR DESCRIPTION
**Pull request by dvoytenko**
Related to #569.

### CODE REVIEW

File: testing/screenshots/screenshot-tests.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+   with the JS/CSS changes in the same PR.
+2. The change is unexpected and it's a regression. The cause should be fixed and testing
+   repeated.
+
+
+## Running tests
+
+All steps assume you are running a web server on localhost:8000. If not - you can change
+the parameters in the below calls using `--host` argument. Refer to the specific `gulp`
+tasks for more info.
```
Comment (erwinmombay) [inline testing/screenshots/screenshot-tests.md]: lets point out they can run `gulp help` for options

**Comment by erwinmombay** [comment]
one small ask, otherwise LGTM.

**Comment by erwinmombay** [comment]
related https://github.com/ampproject/amphtml/pull/731

**Comment by dvoytenko** [comment]
Added `gulp help` definitions and reference to `gulp help`. Also added one more golden screenshot.

**Pull request merged**